src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,180 @@
+// Avatar Component
+
+import { cn } from "@/utils";
+import { cva, VariantProps } from "class-variance-authority";
+import React, { ComponentProps } from "react";
+
+// Avatar
+const AvatarStyles = cva(
+  [
+    // css style
+    "w-full",
+    "h-full",
+    "flex justify-center items-center",
+  ],
+  {
+    variants: {
+      variant: {
+        // variant styles
+      },
+      size: {
+        //size
+        sm: "w-12 h-12",
+        md: "w-14 h-14",
+        lg: "w-16 h-16",
+      },
+    },
+    compoundVariants: [
+      // compound variants
+    ],
+    defaultVariants: {
+      // default variants
+    },
+  }
+);
+
+type CustomAvatarProps = {
+  children?: React.ReactNode;
+  isLoading?: boolean;
+  size?: "sm" | "md" | "lg";
+  src?: string;
+};
+
+type AvatarProps = CustomAvatarProps & VariantProps<typeof AvatarStyles>;
+
+export const Avatar = ({
+  children,
+  isLoading,
+  size,
+  src,
+  ...props
+}: AvatarProps) => {
```

Comment:
Let's use `React.FC` here 👇 

```suggestion
export const Avatar: React.FC<AvatarProps> = ({
  children,
  isLoading,
  size,
  src,
  ...props
}) => {
```